Tools/clinic/clinic.py

```diff
@@ -67,19 +69,26 @@ def __repr__(self):
 
 sig_end_marker = '--'
 
+Appender = Callable[[str], None]
```

Comment:
I am a big fan of using protected names for type alias that are not used in other modules.

Reply:
I don't see the reason to in this case. `clinic.py` is a tool, not a library. And having names starting with underscores everywhere will be pretty unattractive.

Reply:
I guess this is a matter of a personal taste :)